src/v/kafka/server/handlers/handlers.h

```diff
@@ -61,6 +61,12 @@ template<typename... Requests>
 requires(KafkaApiHandler<Requests>, ...)
 using make_request_types = type_list<Requests...>;
 
+/*
+ * This set of handlers defines what the kafka server supports. If you are
+ * adding to this list, then you'll want to also update the list in
+ * kafka/protocol/flex_versions.cc which define properties about the protocol
+ * itself as shared between our client and server.
+ */
```

Comment:
Is it possible to static_assert something that makes this harder to mess up?

Reply:
hmm, i suppose we could assert that the lists of are equal length in practice (though the client, which shares access to this, implements less of the protocol than the server). but i don't think there is anything to mess up from a correctness point of view, rather it's more like, "if you added a server handler and nothing works, you may have forgotten to add to this other list".

Reply:
the other option which is far more practical is to generate more code instead of trying to do all this with template magic. the flip side is that the template magic is kinda cool haha